Makefile

```diff
@@ -20,3 +20,11 @@ copy-travis-files:
 .PHONY:
 test:
 	bundle exec rubocop -R -D
+
+.PHONY:
```

Comment:
No, make won't make files based on target names automatically, but if someone were to manually create a file with the same name as a target (e.g. `test`) then make will stop making that target.